docs/src/pages/[platform]/theming/theme-provider/examples/BasicExample.tsx

```diff
@@ -0,0 +1,11 @@
+import { Button, useTheme } from '@aws-amplify/ui-react';
+
+export const BasicExample = () => {
+  const { tokens } = useTheme();
+
+  return (
+    <Button border={`2px solid ${tokens.colors.blue[80]}`} color="purple.80">
```

Comment:
same here, in the same line, why have `blue[80]` but `purple.80`?

Reply:
For the style props (e.g., `purple.80`), it won't work if you use bracket syntax. And for the theme tokens, it won't work if you use dot syntax.